src/main/java/mate/jdbc/lib/Injector.java

```diff
@@ -4,17 +4,13 @@
 import java.io.IOException;
 import java.lang.reflect.Constructor;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 
 public class Injector {
     private static final Map<String, Injector> injectors = new HashMap<>();
     private final List<Class<?>> classes = new ArrayList<>();
 
-    private Injector(String mainPackageName) {
+    public Injector(String mainPackageName) {
```

Comment:
@EliosPride this is utility class, it must have private constructor. It has static method, so use it like this Injector.getInstance()

Reply:
@Garret-vin ++